pkg/sdk/logging/model/syslogng/config/config.go

```diff
@@ -112,6 +112,21 @@ func configRenderer(in Input) (render.Renderer, error) {
 		in.Logging.Spec.SyslogNGSpec.JSONKeyPrefix = "json" + keyDelim(in.Logging.Spec.SyslogNGSpec.JSONKeyDelimiter)
 	}
 
+	parserDrivers := []render.Renderer{
```

Comment:
I'd name this something more specific to better convey its place
```suggestion
	sourceParsers := []render.Renderer{
```

Reply:
3f31b3d77ca1aa50bb0403e04cd8e4613c30b24f